src/components/assistant_components/AssistantSettings.tsx

```
@@ -397,6 +411,21 @@ const AssistantSettings: React.FC<AssistantSettingsProps> = ({
                 </div>
               </div>
             )}
+
+            {apiType === 'litellm' && (
+              <div className="mt-4">
+                <label className="block text-sm font-medium text-gray-700 dark:text-gray-300 mb-1">
+                  LiteLLM Base URL
+                </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)